wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettings.vue

```diff
@@ -49,14 +50,21 @@ export default {
     });
     document.addEventListener('showSettingsApps', () => this.displayed = true);
     setTimeout( () => {
-      const from = this.getQueryParam('from');
-      if (from) {
+      this.from = this.getQueryParam('from');
+      if (this.from === 'walletApp') {
         const id = this.getQueryParam('id');
         const type = this.getQueryParam('type');
         this.getWallet(id,type);
         window.history.replaceState('wallet', 'My wallet', `${eXo.env.portal.context}/${eXo.env.portal.portalName}/settings`);
         document.location.hash = '#walletSettings';
       }
+      if (this.from === 'space') {
+        const id = this.getQueryParam('id');
+        const type = this.getQueryParam('type');
+        this.getWallet(id,type);
+        window.history.replaceState('wallet', 'My wallet', `${eXo.env.portal.context}/g/:spaces:${eXo.env.portal.spaceGroup}/${eXo.env.portal.spaceName}/settings/settings`);
```

Comment:
the string 'My Wallet' is not correct IMO, and should be replaced with the value of an internationalized one.

Reply:
done!